netebpfext/net_ebpf_ext.c

```diff
@@ -757,37 +772,73 @@ net_ebpf_extension_uninitialize_wfp_components(void)
 {
     size_t index;
     NTSTATUS status;
+    int max_retries = 10;
+    LARGE_INTEGER timeout = {0};
+    timeout.QuadPart = -XMSTO100NS(10000); // 10 seconds
```

Comment:
Since `XMSTO100NS` is only used here, the "MS" units doesn't seem interesting.
Maybe `SECONDSTO100NS(10)`?